<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="vx0">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="masa1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posY0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posX3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="radio2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="masa0">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>210000000</number>
    </property>
    <property name="value">
     <number>50000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posX2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vx4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>-2.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="posY1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="radio4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vy1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>-2.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="masa2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vx3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vx1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="radio3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="masa3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vx2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vy4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="posY3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>-5000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vy2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="masa4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posX1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>-5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posY4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="radio1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="radio0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posY2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>721</width>
      <height>181</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Posicion X inicial</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Posicion Y inicial</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Masa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Radio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Velocidad X inicial</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Velocidad Y inicial</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Sol</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Planeta 1</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>Planeta 2</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>Planeta 3</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>Planeta 4</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="vy0">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="posX4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vy3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="posX0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sistema Gravitacional</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSimulate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener Simulación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSimulate">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Simulación</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="universo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>850</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="randomPlanet">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Planeta</string>
    </property>
   </widget>
   <zorder>tableWidget</zorder>
   <zorder>vx0</zorder>
   <zorder>masa1</zorder>
   <zorder>posY0</zorder>
   <zorder>posX3</zorder>
   <zorder>radio2</zorder>
   <zorder>masa0</zorder>
   <zorder>posX2</zorder>
   <zorder>vx4</zorder>
   <zorder>posY1</zorder>
   <zorder>radio4</zorder>
   <zorder>vy1</zorder>
   <zorder>masa2</zorder>
   <zorder>vx3</zorder>
   <zorder>vx1</zorder>
   <zorder>radio3</zorder>
   <zorder>masa3</zorder>
   <zorder>vx2</zorder>
   <zorder>vy4</zorder>
   <zorder>posY3</zorder>
   <zorder>vy2</zorder>
   <zorder>masa4</zorder>
   <zorder>posX1</zorder>
   <zorder>posY4</zorder>
   <zorder>radio1</zorder>
   <zorder>radio0</zorder>
   <zorder>posY2</zorder>
   <zorder>vy0</zorder>
   <zorder>posX4</zorder>
   <zorder>vy3</zorder>
   <zorder>posX0</zorder>
   <zorder>label</zorder>
   <zorder>stopSimulate</zorder>
   <zorder>startSimulate</zorder>
   <zorder>universo</zorder>
   <zorder>randomPlanet</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
